packages/data-service/src/data-service.spec.ts

```diff
@@ -1646,5 +1646,21 @@ describe('DataService', function () {
         expect(colls).to.deep.eq(['bar']);
       });
     });
+
+    it('allows disabling/enabling the split-client model for CSFLE', function () {
+      const dataService: any = createDataServiceWithMockedClient({});
+      const a = {};
+      const b = {};
+      dataService._crudClient = a;
+      dataService._metadataClient = b;
+
+      expect(dataService._initializedClient('CRUD')).to.equal(a);
+      expect(dataService._initializedClient('META')).to.equal(b);
+
+      dataService.setCSFLEEnabled(false);
+
+      expect(dataService._initializedClient('CRUD')).to.equal(b);
+      expect(dataService._initializedClient('META')).to.equal(b);
```

Comment:
Maybe a bit unnecessary, but if it's easy to add maybe we can add another check to ensure re-enabling it works as expected:

```suggestion
      expect(dataService._initializedClient('META')).to.equal(b);
      
      dataService.setCSFLEEnabled(true);
      
      expect(dataService._initializedClient('CRUD')).to.equal(a);
      expect(dataService._initializedClient('META')).to.equal(b);
```

Reply:
Yeah, makes absolute sense – done!